### ISSUE TITLE
Book that came from Jerusalem

### ISSUE THREAD

**Issue by MarcinKrawczuk** [BetaMasaheft/Documentation#2328]
I have noticed that the texts referred to as "The book that came from Jerusalem" may come with two different incipits:

a) ዛቲ መጽሐፍ ዘወፅአት እምኢየሩሳሌም ከመ ይትናገር ዕበዮሙ ወሢመቶሙ ወክብሮሙ ወትፍሥሕቶሙ ለሚካኤል ወገብርኤል, contained in some versions of Dǝrsāna Mikāʾel

b) መጽሐፍ ዘወፅአት እምኢየሩሳሌም ለመድኃኒተ ነፍስ ወሥጋ በእንተ ጻድቃን ወኃጥአን, contained in Maṣḥafa gǝnzat

Is it the same text and the record LIT3843BookTh can be used for both?

To make things more complicated, in BAVet51 the texts with the same incipit as b) is labelled as LIT4500Mashaf. 

Basically, I need to make a reference to a), hence my question. I will be grateful for your comments.
@thea-m  @CarstenHoffmannMarburg @DenisNosnitsin1970 @MassimoVilla

**Comment by DenisNosnitsin1970** [comment]
I remember vaguely, there is more than one work that has a similar indication to the Jerusalem origin in its incipit.

**Comment by thea-m** [comment]
Thank you for noticing! It seems to require some deeper study, which I right now unfortunately don't have time for - unless someone else does (@CarstenHoffmannMarburg , could it interest you?), maybe, Marcin, you could note your observation in the record  LIT3843BookTh (and use that ID)?

**Comment by CarstenHoffmannMarburg** [comment]
I am busy for the moment as well.

**Comment by DenisNosnitsin1970** [comment]
Now I do not see `LIT3843Bookth` in the database. Does anyone know where it is? It is reserved for a part in "Mashafa genzät" and used in this function in numerous descriptions. I am not sure if I understand right from Marcin's post: is it the same text with different incipit only or different texts that have this title/description, in the beginning, "The Book that came..."?

**Comment by MarcinKrawczuk** [comment]
I would also point out that some printed catalogues (e. g. EMML vol. 8, p. 330 ; Zarzeczny's Tomasi-Lucchesi p. 852) the two books are indexed as separate works.

So far in the database LIT3843BookTh has been used solely for the "Book..." being the part of መጽሐፈ፡ ግንዘት፡. 

The two texts can be compared online:
https://www.bl.uk/manuscripts/Viewer.aspx?ref=or_551_fs001r  (f. 12v-21v)
https://w3id.org/vhmml/readingRoom/view/204830 (f. 113r-115r)

I would be in favor in creating a separate record for the "Book..." from ደርሳነ፡ ሚካኤል፡

**Comment by DenisNosnitsin1970** [comment]
I assume LIT4500Mashaf for BAVet51 is just a mistake, the description six years old, was never changed.

**Comment by MassimoVilla** [comment]
LIT3843BookTh is here: https://betamasaheft.eu/works/LIT3843BookTh/main. The incipit is missing, so it is not associated with any version of the "Book...", but - as Marcin rightly says - it is indicated as part of the መጽሐፈ፡ ግንዘት፡ (`<relation name="saws:formsPartOf" active="LIT3843BookTh" passive="LIT1931Mashaf"/>`). It would be good to add the incipit to this work record and disambiguate it from the other "Book of...".

As far as BAVet51 is concerned, the text is explicitly indicated by Grébaut and Tisserant (p. 211) as "Absolutio super mortuum", just preceded by the title of the "Book..". So, just after the title  ዛቲ፡ መጽሐፍ፡ ዘወፅአት፡ እምኢየሩሳሌም፡ በእንተ፡ ኃጥአን፡ ወአብዳን፨ etc., the text continues with ዛቲ፡ መጽሐፍ፡ ፍትሐት፨ በአፈ፡ ጴጥሮስ፡ ወጳውሎስ፡ ፍቱሐ፡ ረስዮ፡ ለገብርከ፡ እገሌ፨ . The text is therefore given LIT4500Mashaf. Perhaps this description should revised and the text subdivided into two separate items ("Book of...", only the beginning, + Book of the Absolution), with good peace of Grébaut and Tisserant.

I would also be in favour of creating a new record for the "Book of.." in the ደርሳነ፡ ሚካኤል፡
